Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -966,6 +966,7 @@ def _set_items(self, items, axis=1):
         elif not axis:
             model[:] = ["None", "Enumeration", "Attribute names"]
             self.annotation_idx = 2
+            self._update_labels()
```

Comment:
If this is required here, after `annotation_idx` changes, how come it is not required on [line 985 below](https://github.com/biolab/orange3/pull/1701/commits/193b693b6af1554a61ecabf97659120e56657f8c#diff-22fbad01e8c84125308a0a7fe72ba0f1R985)?